charts/argo-workflows/templates/server/server-ingress.yaml

```diff
@@ -36,7 +36,7 @@ spec:
           {{- range $p := $paths }}
           - path: {{ $p }}
             {{- if eq (include "argo-workflows.ingress.apiVersion" $) "networking.k8s.io/v1" }}
-            pathType: Prefix
+            pathType: {{ $pathType }}
```

Comment:
You also need to add this on top of this file (eg. line 6):
```
{{- $pathType := .Values.server.ingress.pathType -}}
```
Ref: https://github.com/argoproj/argo-helm/pull/822/files#diff-cee8e938567dd2c92bf1ac995920fc961d20e4cf5a18b6220fbf869d6bff93e0R6

Reply:
Thank you for catching this!